### PR TITLE
Enable hover for server.xml files

### PR DESCRIPTION
**Pull request by kathrynkodama**
Part of #147 
- Enables hover from LemMinX + Liberty LemMinX extension. 
- Fixes write-unsafe context error when opening external links (e.g. server.xsd) from hover
- Bump IntelliJ Gradle plugin to latest version
- Fixes https://github.com/OpenLiberty/liberty-tools-intellij/issues/242, TimeoutException occurs on first hover in a fresh IntelliJ session.

Hover on server.xml schema elements. Note server.xsd is a click-able link, will open server.xsd file in-editor.
image

Hover on feature names for feature description.
image

Alternatively, users can select the 3 dots -> "Open in documentation tool window" (native IntelliJ mechanism).
image

Now hover text shows up in documentation tool window rather than over the file itself.
image

I did not observe any conflicts between the native IntelliJ hover support in existing xml files, such as pom.xml, and LemMinX.

Signed-off-by: Kathryn Kodama <kathryn.s.kodama@gmail.com>

### CODE REVIEW

**Comment by kathrynkodama** [comment]
Increased the hover timeout from 500ms -> 5s to fix #242 (does not seem to block other UI functions/lead to any UI freezing)

File: src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/operations/hover/LSPTextHover.java [hunk 1]
```diff
@@ -140,11 +140,18 @@ private CompletableFuture<Integer> getCursorOffset(Editor editor) {
     @Nullable
     @Override
     public String generateDoc(PsiElement element, @Nullable PsiElement originalElement) {
-        Editor editor = LSPIJUtils.editorForElement(element);
+        PsiElement elem = element;
+        Editor editor = LSPIJUtils.editorForElement(elem);
+        if (editor == null) {
```
Comment (turkeylurkey) [inline src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/operations/hover/LSPTextHover.java:145]: originalElement could be null so you should check it here.

**Review by turkeylurkey — CHANGES_REQUESTED** [review]

File: src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/operations/hover/LSPTextHover.java (hunk 1 above)
Comment (kathrynkodama) [reply src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/operations/hover/LSPTextHover.java]: Good idea, added null checks for element and originalElement

**Review by turkeylurkey — APPROVED** [review]
Looks good!

**Pull request merged**